vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/component/ExperimentInfoComponent.java

```diff
@@ -0,0 +1,167 @@
+package life.qbic.datamanager.views.projects.project.experiments.experiment.component;
+
+import com.vaadin.flow.component.ComponentEventListener;
+import com.vaadin.flow.component.contextmenu.MenuItem;
+import com.vaadin.flow.component.contextmenu.SubMenu;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.ListItem;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.html.UnorderedList;
+import com.vaadin.flow.component.menubar.MenuBar;
+import com.vaadin.flow.component.menubar.MenuBarVariant;
+import java.io.Serial;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import life.qbic.datamanager.views.general.Card;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Analyte;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Species;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Specimen;
+
+/**
+ * <b>Sample Origin Component</b>
+ *
+ * <p>A component that nicely renders sample origin information of an experiment</p>
+ *
+ * @since 1.0.0
+ */
+public class ExperimentInfoComponent extends Card {
```

Comment:
The Javadocs and naming schema don't match here 
```suggestion
public class SampleOriginComponent extends Card {
```

Reply:
as discussed in Slack, the name sample origin is misleading, thus I changed it.